clients/gtest/auxiliary_gtest.yaml

```diff
@@ -84,6 +84,11 @@ Tests:
   function:
     - aux_get_sol_with_null_biasaddr: *hpa_half_precision
 
+- name: aux_get_sol_with_zero_alpha_null_a_b
+  category: pre_checkin
+  function:
```

Comment:
The original request is for cpp path.
It is better to add test for cpp path as well.